src/components/FilterElement/FilterElement.tsx

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { ChangeEvent, useEffect, useRef, useState } from 'react';
+import { Form, Spinner } from 'react-bootstrap';
+import "react-datepicker/dist/react-datepicker.css";
+import './FilterElement.css';
+import api from '../../api/ApiHelper';
+import DatePicker from "react-datepicker";
+
+// has to be redefined because global types from react-app-env are not accessable
```

Comment:
are you sure? Why not?

Reply:
typescript doesn't deliver the global types definition file.